packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -118,6 +124,14 @@ import {
 } from '../../../../stores/shared/ModelUtil.js';
 import { LEGEND_STUDIO_APPLICATION_NAVIGATION_CONTEXT_KEY } from '../../../../stores/LegendStudioApplicationNavigationContext.js';
 
+export interface ClassPropertyDragSource {
+  columnState: Property;
```

Comment:
I prefer to move these two classes to ClassState.ts

Reply:
synced up with An so it will be ok to just leave here for now